test/compress/varify.js

```diff
@@ -354,6 +354,92 @@ forin_let_2: {
     node_version: ">=6"
 }
 
+forin_let_3: {
+    options = {
+        toplevel: true,
+        varify: true,
+    }
+    input: {
+        "use strict";
+        var o = { foo: 1, bar: 2 };
+        for (let i in o) {
+            console.log(i);
+        }
+        for (let j in o)
+            setTimeout(() => console.log(j), 0);
+        for (let k in o)
+            setTimeout(function() {
+                console.log(k);
+            }, 0);
+    }
+    expect: {
+        "use strict";
+        var o = { foo: 1, bar: 2 };
+        for (var i in o)
+            console.log(i);
+        for (let j in o)
+            setTimeout(() => console.log(j), 0);
+        for (let k in o)
+            setTimeout(function() {
+                console.log(k);
+            }, 0);
+    }
+    expect_stdout: [
+        "foo",
+        "bar",
+        "foo",
+        "bar",
+        "foo",
+        "bar",
+    ]
+    node_version: ">=4"
+}
+
+forin_let_4: {
```

Comment:
No for-in in this test. Can you mix `let` and `const` use in the example?

Reply:
Done in https://github.com/mishoo/UglifyJS/pull/4719/commits/e3ad69aeffd93e10de683c8b3727b7ed9afd262e